bitcoin/src/blockdata/script/instruction.rs

```diff
@@ -162,6 +160,16 @@ pub(super) enum PushDataLenLen {
     Four = 4,
 }
```

Comment:
OMG, we already had this enum (IIRC added by me) we just weren't using it consistently. Why not just reuse it?

Reply:
I couldn't decide if it was better to keep two enums, kind of private, or provide one public one, so I went with the two. `PushDataLenLen` is already a tough name to process, let alone make a new name that covers this use case as well as `ReadUintSize`. Anyway, I can think a bit harder today.

Reply:
Can this be resolved as a follow up please, this PR is blocking #3194

Reply:
Name etc doesn't matter, it lives in `internals` anyway. Seems quite a bit of noise to me adding a whole new enum and then removing it and renaming it even if we're doing this temporarily. It makes `git blame` more annoying for instance.

Reply:
Ok, I renamed it to `UintSize`, removed `PushDataLenLen`, and changed the new `internals` module to be called `script`.